src/Microsoft.OData.Core/ODataInnerError.cs

```diff
@@ -93,5 +133,34 @@ internal string ToJson()
                 this.StackTrace == null ? "" : JsonValueUtils.GetEscapedJsonString(this.StackTrace),
                 this.InnerError == null ? "{}" : this.InnerError.ToJson());
         }
-}
+
+        private string LookUpStringValue(string v)
+        {
+            if (Properties == null)
+            {
+                return string.Empty;
+            }

```

Comment:
Properties should never be null?

Reply:
What if someone uses the default constructor. 

---
In reply to: [320874186](https://github.com/OData/odata.net/pull/1523#discussion_r320874186) [](ancestors = 320874186)